mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -271,6 +271,30 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 										Description: `Rate limit key name applicable only for the following key types: HTTP_HEADER -- Name of the HTTP header whose value is taken as the key value. HTTP_COOKIE -- Name of the HTTP cookie whose value is taken as the key value.`,
 									},
 
+									<% unless version == 'ga' -%>
+									"enforce_on_key_configs": {
+										Type:        schema.TypeList,
+										Description: `Enforce On Key Config of this security policy`,
+										Optional:    true,
+										MaxItems:    3,
```

Comment:
Normally, we will leave the validation on server side in case the validation will change, unless the users requested it.

Reply:
It's an internal Google request to set this new providers. I don't know if a answer your question correctly.